### PR TITLE
port roadmap doc from issue to repo

### LINKED ISSUE #250: cut version 1.0 of k2
determine what the necessary features are for k2 v1.0 and then cut a release.  if the new CI system is running, build the process there.  if not, build the release process in the old CI system.

### PR DESCRIPTION
**Pull request by coffeepac**
closes #250 as this is a better way to track our roadmap

### CODE REVIEW

**Comment by DStorck** [comment]
are we still referring to the project as K2, or should all references to k2 be changed to kraken?

File: roadmap.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+
+# Roadmap for Kraken2 and related projects
+Kraken2 (K2) is the most important piece of our tool chain and we let it drive the roadmaps for the other proects. If those other projects require significant effort we will start a stub mention here so we address it during normal project planning and grooming.
```
Comment (leahnp) [inline roadmap.md]: *projects (spelling)

**Comment by leahnp** [comment]
To confirm this is part of external communication?  I am trying to understand who the 'reader' is.

**Comment by leahnp** [comment]
It's looking good!

**Comment by coffeepac** [comment]
@leahnp reader is anyone who wants to know what our plan is.  this include internal samsung folks and the community at large.

File: roadmap.md (hunk 1 above)
Comment (coffeepac) [inline roadmap.md]: thanks!  the 'j' key on my mac only works like half the time.  it was really annoying when I was having a conversation about jenkins jobs.

**Comment by countspongebob** [comment]
Some things to get done, some of which not directly codeslingy but need to be on the roadmap somewhere

- deprecation of k2 naming, renaming k2 to kraken
- scrub through community docs to provide the right naming, pointers
- Community kraken demo
- Staffing kickoff of the kraken community channel.- Logging architecture as a blog by one of the team members (do we need some bot work to ensure we aren't missing traffic?)
- Connecting specific monitored items (SLIs) to an alert
- Alert delivery. Suggest first target would be some kind of healthcheck "is cluster alive". Possibly the heptio cluster inspector might be a good choice.
- Medium / blog post on Kraken, kubeadm
- Contributing results to the community test results dashboard. Possibly the best single marketing exposure project.
- Room visible dashboard for kraken CI? Discuss.

**Comment by coffeepac** [comment]
@countspongebob added comments to the roadmap to cover, I think, all of your comments OR dropped an edit onto your comment except for the visible dashboard.

Visible dashboard:  I don't see it as real useful, it just drops into the background like art and I don't notice it much.  If we want one the default jenkins UI for the pipeline is good.

**Pull request merged**